src/ui/QuestionsAnswering/QuestionsAnswering.ts

```diff
@@ -0,0 +1,39 @@
+import { exportGlobally } from '../../GlobalExports';
+import { Component } from '../Base/Component';
+import { IComponentBindings } from '../Base/ComponentBindings';
+import { QueryEvents, Initialization, $$ } from '../../Core';
+import { IQuerySuccessEventArgs } from '../../events/QueryEvents';
+import { IQueryQuestionAnswer } from '../../rest/QueryAnswers';
+
+export interface IQuestionsAnsweringOptions {}
+
+export class QuestionsAnswering extends Component {
```

Comment:
QuestionsAnswering or rather QuestionAnswering? It would be more consistent if Question was always singular for those classes/interfaces, except if at some point you have a list of Question

Reply:
I was actually trying to be consistent with the name of the feature as referred to in the Slack channel. However, I do see that in the google doc it's singular. So I imagine it may be up to whether this component shows one or multiple answers.

Currently, it's hard to tell whether there will be one or multiple answers. I'll leave this plural until we know for sure.

Reply:
I like your suggestions a lot better. I'll change it to `SmartSnippet`, I find it very distinguishing.